Dockerfile

```diff
@@ -61,32 +61,32 @@ RUN touch chimp_chomp/src/lib.rs \
 # Collate dynamically linked shared objects for chimp_chomp
 RUN mkdir /chimp_chomp_libraries \
     && cp \
-        $(ldd /app/target/release/chimp_chomp | grep -o '/.*\.so\S*') \
-        /app/target/release/libonnxruntime.so.* \
-        /chimp_chomp_libraries
+    $(ldd /app/target/release/chimp_chomp | grep -o '/.*\.so\S*') \
+    /app/target/release/libonnxruntime.so.* \
+    /chimp_chomp_libraries
```

Comment:
Presumably this a difference between our formatters, but would rather this was kept as before